src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -244,34 +241,25 @@ public class PublicationService : IPublicationService
                         publication.Published = DateTime.UtcNow;
                         await _context.SaveChangesAsync();
 
-                        await UpdateCachedTaxonomyBlobs();
-
-                        await _publicBlobCacheService.DeleteItem(new PublicationCacheKey(publication.Slug));
+                        await _methodologyCacheService.UpdateSummariesTree();
+                        await _themeCacheService.UpdatePublicationTree();
+                        await _publicationCacheService.UpdatePublication(publication.Slug);
 
-                        await DeleteCachedSupersededPublicationBlobs(publication);
+                        await UpdateCachedSupersededPublications(publication);
```

Comment:
I was wondering if this code could eventually move into `_publicationCacheService.UpdatePublication()` itself.  I guess that kind of refactoring would happen at the point where we change the communication mechanism between Publisher and the API projects (if we do end up doing that).

Reply:
Ah good point, thanks. I could look to see if the work I'm doing for EES-3645 would be a good place to do this because I can see similar code in multiple places across Admin and Publisher.